test-projects/simple-database/project.visivo.yml

```diff
@@ -37,6 +37,17 @@ targets:
 models:
   - name: test table local
     sql: "SELECT * FROM test_table"
+  - name: csv
+    args:
```

Comment:
Ohhh it's arguments to a one liner. I was thinking that each item in the args list would be a command to be run in succession

Reply:
Yes, this is a pretty common method.  It prevents confusing quotation issues.